test/unit/functions.js

```diff
@@ -68,4 +68,15 @@ function test_trap(t, params, subjects ) {
 
 }
 
+test('rastrigin', function(t) {
+	 for (var i in dimensions) {
```

Comment:
The body of a for in should be wrapped in an if statement to filter unwanted properties from the prototype.